app_flutter/lib/widgets/task_overlay.dart

```diff
@@ -291,7 +290,6 @@ class TaskOverlayContents extends StatelessWidget {
                               'Task was run outside of devicelab',
                               style: Theme.of(context).textTheme.bodyText2,
                             ),
-                          if (QualifiedTask.fromTask(task).isDevicelab) TaskAttemptSummary(task: task),
                           if (QualifiedTask.fromTask(task).isLuci) LuciTaskAttemptSummary(task: task),
```

Comment:
Can we rename `LuciTaskAttemptSummary` to `TaskAttemptSummary` now?

Reply:
I'm leaning to leaving it as is for simplicities sake, unless there is a compelling reason to rename.

Reply:
With the code cleanup from last week all tasks are `LuciTask`, so I think it adds TD to leave this naming here (how does `LuciTask` differ from `Task`). I'll defer to @godofredoc for what we should do.

Reply:
With your changes to remove the stage concept, is QualifiedTask going away as well?

Reply:
I removed `Stage` from the backend (+ whatever was needed in the frontend). Feel free to remove it if it no longer makes sense.

Reply:
.isLuci is based on the stage thats set when creating QualifiedTask. That still seems to be around after your change?

Reply:
Yes, that's a frontend relic. My PR should have been labeled "remove stage from backend"